docs/swagger.yaml

```diff
@@ -82,6 +82,96 @@ paths:
       tags:
         - Merchant redirect QR
     parameters: []
+    get:
+      summary: Get all QRs
+      operationId: GetAllMerchantRedirectQrs
+      responses:
+        '200':
+          description: OK
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/MerchantRedirectQrResponseGetAll'
+        '400':
+          description: Bad Request
+        '401':
+          description: Unauthorized
+      description: Get all QRs created for one saleunit
+      tags:
+        - Merchant redirect QR
+      parameters:
+        - $ref: '#/components/parameters/Authorization'
+        - $ref: '#/components/parameters/Ocp-Apim-Subscription-Key'
+  '/v1/merchantRedirect/{id}':
+    put:
+      summary: Update QR redirectUrl
+      operationId: UpdateMerchantRedirectUrl
+      responses:
+        '200':
+          description: OK
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/MerchantRedirectQrResponse'
+        '401':
+          description: Unauthorized
+        '404':
+          description: Not Found
+      description: Update redirectUrl
+      tags:
+        - Merchant redirect QR
+      requestBody:
+        content:
+          application/json:
+            schema:
+              $ref: '#/components/schemas/MerchantRedirectQrUpdateRequest'
+      parameters:
+        - $ref: '#/components/parameters/Accept'
+        - $ref: '#/components/parameters/Authorization'
+        - $ref: '#/components/parameters/Ocp-Apim-Subscription-Key'
+    delete:
+      summary: Delete QR
+      operationId: DeleteMerchantRedirectQr
+      responses:
+        '200':
+          description: OK
+        '401':
+          description: Unauthorized
+        '404':
+          description: Not Found
+      description: Delete QR
+      tags:
+        - Merchant redirect QR
+      parameters:
+        - $ref: '#/components/parameters/Authorization'
+        - $ref: '#/components/parameters/Ocp-Apim-Subscription-Key'
+    parameters:
+      - schema:
+          type: string
+        name: id
+        in: path
+        required: true
+        description: id for QR
+    get:
+      summary: Get all saleunit QRs
```

Comment:
"Get all Merchant Redirect QRs"

Reply:
This wasnt added on all endpoints @kaffikaffi 
![image](https://user-images.githubusercontent.com/47554984/152022695-4f315d02-2dd8-4a62-8338-a1ff36860a5c.png)
its missing on delete and Get endpoints